src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -31,29 +32,17 @@ public interface IInvocationExpression : IExpression
         /// <summary>
         /// 'This' or 'Me' argument to be supplied to the method.
         /// </summary>
-        IExpression Instance { get; }
+        IOperation Instance { get; }
```

Comment:
How about "'This' or 'Me' instance to be supplied to the method, or null if the method is static."?